### ISSUE TITLE
[BUG] No "Are you sure?" prompt when cancelling edits

### ISSUE THREAD

**Issue by bobf32** [PhilippC/keepass2android#2733]
### Checks

- [X] I have read the FAQ section, searched the open issues, and still think this is a new bug.

### Describe the bug you encountered:

When I make some edits to a password entry and then hit the cancel button I am sent immediately back to the entry list.

### Describe what you expected to happen:

When an entry is "dirty" i.e. edits have been made, I would expect some kind of prompt asking me if I am sure I want to discard my edits, before the edit operation is aborted.

### What version of Keepass2Android are you using?

1.11-r0

### Which version of Android are you on?

13

**Comment by Gian-Fr** [comment]
If you mean the button in the upper right, I think that's intentional.

**Comment by bobf32** [comment]
> If you mean the button in the upper right, I think that's intentional.

Yes, the cross in the top right is what I'm referring to. Why would this be intentional? Surely closing the window should not go unremarked if there are pending edits.

Also, why do we have a back button and a cancel button? Seems slightly overkill. And just confusing that back does prompt on dirty edits but cancel doesn't. I would have thought one action was needed, and that it warned when any unsaved edits were made.
